src/components/AppHeader/AppHeader.tsx

```diff
@@ -96,11 +96,24 @@ export const AppHeader: FunctionComponent<any> = (props) => {
             </Tooltip>
           </PopoverTrigger>
           <PopoverSurface aria-label="Your roles">
-            <ul>
-              {userContext.roles?.map((role) => (
-                <li key={role}>{role}</li>
-              ))}
-            </ul>
+            {
+              /** If the user has role(s), list them */
+              userContext.roles && userContext.roles.length > 0 && (
+                <ul>
+                  {userContext.roles?.map((role) => (
+                    <li key={role}>{role}</li>
+                  ))}
+                </ul>
+              )
+            }
+            {
+              /** If the user has no privleged role(s), just state standard account */
+              userContext.roles && userContext.roles.length === 0 && (
+                <ul>
+                  <li>Standard user account</li>
+                </ul>
+              )
+            }
```

Comment:
I tried it that way originally, but VSCode was complaining it could possibly be undefined.  Tried again this morning, and getting same thing.  Am I missing something?